infrastructure/vm/vagrantfile-so/publish.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-vagrant cloud publish edi33416/so-ubuntu-18.04 1.1.0 virtualbox so-ubuntu-18-04.box -d "Ubuntu 18.04 based on bento/ubuntu-18.04 and provisioned with Ansible" --version-description "Latest stable version" --release --short-description "SO VM"
+vagrant cloud publish edi33416/so-ubuntu-20.04 1.1.0 virtualbox so-ubuntu-20-04.box -d "Ubuntu 20.04 based on bento/ubuntu-20.04 and provisioned with Ansible" --version-description "Latest stable version" --release --short-description "SO VM"
```

Comment:
```suggestion
vagrant cloud publish edi33416/so-ubuntu-20.04 1.3.0 virtualbox so-ubuntu-20-04.box -d "Ubuntu 20.04 based on bento/ubuntu-20.04 and provisioned with Ansible" --version-description "Latest stable version" --release --short-description "SO VM"
```

Reply:
Bump the version to 1.3.0, as there is 1.2.0 is the current [release](https://app.vagrantup.com/edi33416/boxes/so-ubuntu-18.04/versions/1.2.0)